modules/balance-ignored-labels-autoscaler-hcp.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-machine-config.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="balance-ignored-labels-autoscaler-hcp_{context}"]
+= Balancing ignored labels in a hosted cluster
+
+You can use the `node.group.balancing.ignored` label to evenly distribute the machines provisioned by different node pools with the same balancing ignored labels.
+
+.Prerequisites
+
+* You have created the `HostedCluster` and `NodePool` resources.
+
+.Procedure
+
+. Add the `node.group.balancing.ignored` label to each of the relevant node pools by using the same label value. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> \
+  nodepool <node_pool_name> \
+  --type=merge \
+  --patch='{"spec": {"nodeLabels": {"node.group.balancing.ignored": "<label_name>"}}}'
+----
+
+. Enable cluster autoscaling for the hosted cluster, which sets the scaling parameters and the default expander. Run the following command:
++
+[source,terminal]
+----
+$ oc patch -n <hosted_cluster_namespace> \
+  hostedcluster<hosted_cluster_name> \
+  --type=merge \
+  --patch='{"spec": {"autoscaling": {"scaling": "ScaleUpAndScaleDown", "maxPodGracePeriod": 60, "maxFreeDifferenceRatioPercent": 50, "scaleDown": {"delayAfterAddSeconds": 60, "delayAfterDeleteSeconds": 60, "delayAfterFailureSeconds": 60, "unneededDurationSeconds": 60, "utilizationThresholdPercent": 50}, "expanders": ["Random"]}}}'
```

Comment:
I used several `seconds` fields during the tests to speed up the scaling down, but maybe we can ignore that in the docs example so the default values will be used ? not sure this will give users any unnecessary hints.